bin/webpack-dev-server

```diff
@@ -4,16 +4,15 @@ ENV["RAILS_ENV"] ||= ENV["RACK_ENV"] || "development"
 ENV["NODE_ENV"]  ||= "development"
 
 require "pathname"
-
-APP_ROOT = File.expand_path("..", __dir__)
-
-ENV["BUNDLE_GEMFILE"] ||= File.expand_path("Gemfile", APP_ROOT)
+ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile", Pathname.new(__FILE__).realpath)
```

Comment:
Why this change? it's not exactly the same?

Reply:
Yes, but this is auto generated by the webpacker installer.